cmd/describe/breakglasscredential/cmd.go

```diff
@@ -124,6 +124,9 @@ func runWithRuntime(r *rosa.Runtime, cmd *cobra.Command, argv []string) error {
 	}
 
 	if getKubeconfig {
+		if breakGlassCredentialConfig.Kubeconfig() == "" {
+			r.Reporter.Infof("The credential is not ready yet. Please wait a few minutes for it to be fully ready.")
```

Comment:
minor, but we could return directly after, since in any case we are printing an empty string

Reply:
@andreadecorte Updated